app/controllers/projects_controller.rb

```diff
@@ -30,9 +31,36 @@ def project
     @_project ||= Project.find(params[:id])
   end
 
+  def project_content_items_to_display
+    @_project_content_items_to_display = begin
+      items = project.content_items
+
+      tagged_state_filter = filter_params[:taggedState]
```

Comment:
Yes, but it only ended up that way as I think I'm more use to using that style for ids on the DOM nodes. I've changed this to use underscores instead, as I think this would be more consistent with the Ruby code, which is what I'm guessing is the reason behind your question.